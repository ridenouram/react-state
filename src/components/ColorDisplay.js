import React from 'react';
import PropTypes from 'prop-types';

export default function ColorDisplay({ color }) {
  return (
    <div style={{ backgroundColor: color, width: '100px', height: '100px' }}></div>
  );
}

ColorDisplay.propTypes = {
  color: PropTypes.string.isRequired
};
