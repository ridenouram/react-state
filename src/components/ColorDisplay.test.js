import React from 'react';
import { shallow } from 'enzyme';
import ColorDisplay from './ColorDisplay';

describe('ColorDisplay component', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<ColorDisplay color="red" />);
    expect(wrapper).toMatchSnapshot();
  });
});

