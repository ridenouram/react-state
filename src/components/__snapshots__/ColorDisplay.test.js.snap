// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorDisplay component matches snapshot 1`] = `
<div
  style={
    Object {
      "backgroundColor": "red",
      "height": "100px",
      "width": "100px",
    }
  }
/>
`;
